commands/up.go

```diff
@@ -15,11 +16,11 @@ func Up(session *discordgo.Session, interaction *discordgo.InteractionCreate) {
 
 	if len(interaction.ApplicationCommandData().Options) < 1 {
 		captchaKey := service.RandomString(4, "1234567890")
-		captchaImage := service.GenerateCaptcha(captchaKey)
+		captchaImage, _ := service.GenerateCaptcha(captchaKey)
```

Comment:
why you return error and don't use it?

Reply:
I think the reason is because @Dellyare wanted to make error handling later. I just left it for the future